jme3-android-examples/jme3-android-tests/stack-alloc/build/.transforms/ab8191ea4370caf26f070fcbd7827d08.bin

```diff
@@ -0,0 +1 @@
+o/jetified-stack-alloc
```

Comment:
what is jetified-stack-alloc and why  `jetified-stack-alloc/classes.dex` is committed to repo?

Reply:
.dex extension files are the dalvik vm executable files, they are auto-generated by android studio here as a conversion from .class(from the jar file) to .dex to be runnable inside an apk package, that's why it needs to be imported as android module & not copied/pasted as jar.

https://source.android.com/devices/tech/dalvik/dex-format

Reply:
I mean if they are automatically generated when building the project then I think you should not include any of those (.bin, .dex, jetified jars) in your PR. Am I right?

Reply:
If i use the import module feature of jetifier, then they are extracts & remodifications of jars, that must be added by default or you will get a broken project ofc, but if i tried again the gradle jar, and we reached a fix then this would be done(conversion) of files to dex format only inside an apk file using the ```adb``` only & it won't be included in the PR.